docs/advanced/api/authentication.rst

```diff
@@ -0,0 +1,63 @@
+==============
+Authentication
+==============
+
+The Flocker REST API uses TLS certification to secure and authenticate requests.
+This ensures an API request is both encrypted and verified to have come from an authorised user, while the corresponding response is verified to have come from the genuine cluster control service.
+
+Unlike regular HTTPS, Flocker uses TLS not just for establishing a secure connection between an API user and the cluster's control service, but also for identity verification; that is, client certification is used in place of a username and password to identify an API user to the cluster.
+
+Therefore to grant a user access to your cluster's REST API, you will need to use the ``flocker-ca`` tool installed as part of the ``flocker-cli`` package to generate a certificate and private key that is then given to the API end user.
+When you installed the ``flocker-node`` package on each of your cluster nodes, you should have already used the ``flocker-ca`` tool to generate a root certificate authority identifying your cluster.
+If you have not already followed these steps, please see the `flocker-node installation instructions <../indepth/installation>`.
+
+Generate an API user certificate
+================================
+
+The CLI package includes the ``flocker-ca`` program which is used to generate certificate and key files.
+
+.. code-block:: console
+
+    $ flocker-ca --help
+
+    Usage: flocker-ca <command> [OPTIONS]
+    Options:
+          --version  Print the program's version and exit.
+          --help     Display this help and exit.
+      -v, --verbose  Turn on verbose logging.
+
+    flocker-ca is used to create TLS certificates.
+    The certificates are used to identify the control service, nodes
+    and API clients within a Flocker cluster.
+    Commands:
+        initialize                      Initialize a certificate authority in the current
+                                        working directory.
+        create-control-certificate      Create a certificate for the control service.
+        create-node-certificate         Create a certificate for a node.
+        create-api-certificate          Create a certificate for an API user.
+
+You will need to run the ``create-api-certificate`` command from the same directory containing the certificate authority files generated when you first installed the cluster.
```

Comment:
Use the full command name (``flocker-ca create-api-certificate``) everywhere (you'd expect to see Docker refer to ``docker run`` rather than the ``run`` command).